ui/round/src/keyboardMove.ts

```diff
@@ -1,11 +1,18 @@
 import { h } from 'snabbdom'
+import { sanToRole } from 'chess'
 import * as cg from 'chessground/types';
 import { Step, Redraw } from './interfaces';
 import RoundController from './ctrl';
+import { valid as crazyValid } from './crazy/crazyCtrl';
 
 export type KeyboardMoveHandler = (fen: Fen, dests?: cg.Dests) => void;
 
+interface SanMap {
+  [key: string]: cg.Role;
+}
+
 export interface KeyboardMove {
+  drop(orig: cg.Key, piece: string): void
```

Comment:
Note to self: Can't use cg.Role, because sanToRole is not available in the plugin.